src/bundle/linux/deb_bundle.rs

```diff
@@ -109,7 +109,12 @@ fn generate_control_file(
     )?;
     writeln!(&mut file, "Version: {}", settings.version_string())?;
     writeln!(&mut file, "Architecture: {arch}")?;
-    writeln!(&mut file, "Installed-Size: {}", total_dir_size(data_dir)?)?;
+    // deb Installed-Size is size in bytes / 1024
```

Comment:
May as well include the URL citation in this comment that you linked to in the issue? (https://www.debian.org/doc/debian-policy/ch-controlfields.html#s-f-installed-size)

Reply:
Sure!